Makefile

```diff
@@ -125,7 +125,7 @@ setup-e2e-tests: setup-test-infra venv ## Start kind cluster and set up end to e
 	@$(ACTIVATE_VENV) && $(CI_CMD) setup-e2e-tests "$(COMMIT_SHA)" "$(DEV_IMAGE_REPO)" "$(GIT_TAG)" "$(TEST_IMAGE_NAME)"
 
 test-e2e: setup-test-infra venv ## Run end to end tests (assuming cluster is running and set up has been run)
-	@$(ACTIVATE_VENV) && $(CI_CMD) e2e-tests
+	@$(ACTIVATE_VENV) && $(CI_CMD) test-cli
```

Comment:
I don't see a 'test-cli' target in this Makefile

Reply:
It's in test-infra, in common_tasks:

```
test-cli() {
  hash faker || python -m pip install -q faker
  hash anchore-cli || python -m pip install -q anchorecli

  print_colorized INFO "Running CLI tests."; echo

  python anchore-ci/cli_driver.py

  print_colorized INFO "Finished running CLI tests."; echo
}
```